internal/network/utils.go

```diff
@@ -374,6 +380,12 @@ func UpdateVipsTables(db *gorm.DB, cluster *common.Cluster, apiVipUpdated bool,
 
 	if apiVipUpdated {
 		if err := db.Transaction(func(tx *gorm.DB) error {
+			if err := db.Model(&models.Cluster{}).Where("id = ?", *cluster.ID).
```

Comment:
if you are using update do you still need the delete below? 
same comment for the `ingress_vip`

Reply:
actually the delete is for the table, ignore